src/renderer/screens/Editor/Sidebar/LeaderKeys.js

```diff
@@ -15,13 +15,17 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import React from "react";
+import React, { useContext, useEffect, useState } from "react";
```

Comment:
Do we really need these extra imports?

Reply:
We do not, that's a leftover from my first iteration.